recipes/dsp-filters/all/conandata.yml

```
@@ -0,0 +1,12 @@
+sources:
+  "20170309":
```

Thanks a lot for the contribution. One thing about the version is that we tipically add the cci. prefix to distinguish as a virtual version provided by CCI (please check https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#version)

```suggestion
  "cci.20170309":
```

---

Ahhhhhhhhh. I have actually been looking for information about whether I should prefix with "cci." as I have observed with other packages. Thanks for the link!